asn1/src/main.rs

```diff
@@ -1,32 +1,58 @@
 use honggfuzz::fuzz;
-use simple_asn1::*;
-use asn1::*;
+
+// This fuzzer tests the following DER strcuture:
```

Comment:
```suggestion
// This fuzzer tests the following DER structure:
```
fix the typo and we´re done!